### PR TITLE
Add pushing to production info

### PR DESCRIPTION
**Pull request by ingkebil**

### CODE REVIEW

File: conda/conda_conventions.md [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
-      - env_type: Allowed environment types are **D** (develop) or **P** (production).
-      - creation_date: YYMMDD.
+   - Archive: `[env_type=A]_[logical_name]_[timestamp]_[signature]
+      - env_type: Allowed environment types are **D** (Develop), **P** (Production), or **A** (Archive).
+      - creation_date: %y%m%d
+      - timestamp: %y%m%d %H%M%S.
       - logical_name: Whatever makes sense.
-      - signature: Something to show who created the environment. Use the two- or three letter name akronyms assigned to you.
+      - signature: Something to show who created the environment. Use the two- or three letter name acronyms assigned to you.
 - List your production environment and dependent processes at the bottom of this page.
```
Comment (henrikstranneheim) [inline conda/conda_conventions.md]: We should remove the "bottom of this page" and add it here instead: https://github.com/Clinical-Genomics/servers

under conda envs

File: development/production.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# How to push to production?
+
+... and rollback if things go haywire!
+
+## Snapshot!
+
+Take a snapshot of hte current conda env!
```
Comment (henrikstranneheim) [inline development/production.md]: hte -> the

File: development/production.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+```savetheconda prod170926```
```
Comment (henrikstranneheim) [inline development/production.md]: Would be nice with a "$" to know that this is the command.

**Comment by ingkebil** [comment]
I'm going to be impolite and merge already :) Please poke if you don't agree!

**Pull request merged**